### ISSUE TITLE
svg in file uploader should not be focusable

### ISSUE THREAD

**Issue by makandre** [IBM/carbon-components-angular#2233]
## Detailed description

> Describe in detail the issue you're having.

The error icon in the file uploader is flagged by accessibility checker as being focusable when it has `aria-hidden="true"` set. This is because it also has `tabindex="0"`

![image](https://user-images.githubusercontent.com/17596240/187983132-d86c5518-f532-4c15-82f3-f3e9f1f5fb4c.png)

<img width="802" alt="image" src="https://user-images.githubusercontent.com/17596240/187983497-eb67211b-d866-485d-b168-eda8086546ea.png">

> Is this a feature request (new component, new icon), a bug, or a general issue?

Accessibility bug

> Is this issue related to a specific component?

File Uploader

> What did you expect to happen? What happened instead? What would you like to see changed?

The svg should not be focusable or `aria-hidden` should be false

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

`"carbon-components-angular": "^4.59.12"`

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

Component of IBM ADDI

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

* Screenshots or code
* Notes
